<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>331</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt;Задержка перед каждым&lt;br/&gt;постом в секундах.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt;Приблизительное время &lt;br/&gt;до окончания&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите текс сообщения</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Альбомы новый Славик</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Альбомы старый Славик</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_movie">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Альбом с чехлами</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Кнопки</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionStart_messages"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay_mess"/>
   <addaction name="separator"/>
   <addaction name="actionPause_mess"/>
   <addaction name="separator"/>
   <addaction name="actionStop_mess"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="vk_res.qrc">
     <normaloff>:/image/images/login.png</normaloff>:/image/images/login.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionStart_messages">
   <property name="icon">
    <iconset resource="vk_res.qrc">
     <normaloff>:/image/images/trig2.png</normaloff>:/image/images/trig2.png</iconset>
   </property>
   <property name="text">
    <string>Start messages</string>
   </property>
  </action>
  <action name="actionPlay_mess">
   <property name="icon">
    <iconset resource="vk_res.qrc">
     <normaloff>:/image/images/play.png</normaloff>:/image/images/play.png</iconset>
   </property>
   <property name="text">
    <string>play_mess</string>
   </property>
  </action>
  <action name="actionPause_mess">
   <property name="icon">
    <iconset resource="vk_res.qrc">
     <normaloff>:/image/images/pause.png</normaloff>:/image/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>pause_mess</string>
   </property>
  </action>
  <action name="actionStop_mess">
   <property name="icon">
    <iconset resource="vk_res.qrc">
     <normaloff>:/image/images/stop.png</normaloff>:/image/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>stop_mess</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vk_res.qrc"/>
 </resources>
 <connections/>
</ui>
